### ISSUE TITLE
Forum topic switches pages on scroll

### ISSUE THREAD

**Issue by wilco375** [csvalpha/amber-ui#543]
When on a phone, a forum topic switches pages when scrolling down

**Comment by guidojw** [comment]
Can't replicate on my iPhone

**Comment by DrumsnChocolate** [comment]
this may have to do with how sensitive the swiping component is

**Comment by wilco375** [comment]
Yeah it scrolls to the next page if a horizontal scroll threshold is met I think, regardless of the vertical offset. So if you go a bit too much to the left or right when you swipe down it switches to another page. Probably good to add a check so that the page isn't changed when over a certain vertical threshold

**Comment by DrumsnChocolate** [comment]
I was thinking more like a minimal threshold in swiping direction, and then like at least a 4:1 ratio of distance traveled.
